generatebundlefile/hack/release_staging.sh

```diff
@@ -0,0 +1,100 @@
+# limitations under the License.
+
+
+set -e
+set -x
+set -o pipefail
+
+export LANG=C.UTF-8
+
+cat << EOF > configfile
+[profile packages]
+role_arn=$PACKAGES_ARTIFACT_DEPLOYMENT_ROLE
+region=us-west-2
+credential_source=EcsContainer
+EOF
+
+# Release Package Images to Packages Artifact account
+BASE_DIRECTORY=$(git rev-parse --show-toplevel)
+export AWS_CONFIG_FILE=${BASE_DIRECTORY}/generatebundlefile/configfile
+export PROFILE=packages
+AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+. "${BASE_DIRECTORY}/generatebundlefile/hack/common.sh"
+ORAS_BIN=${BASE_DIRECTORY}/bin/oras
+
+make build
+chmod +x ${BASE_DIRECTORY}/generatebundlefile/bin/generatebundlefile
+
+aws ecr get-login-password --region us-west-2 | HELM_EXPERIMENTAL_OCI=1 helm registry login --username AWS --password-stdin ${AWS_ACCOUNT_ID}.dkr.ecr.us-west-2.amazonaws.com
+aws ecr-public get-login-password --region us-east-1 | HELM_EXPERIMENTAL_OCI=1 helm registry login --username AWS --password-stdin public.ecr.aws
+
+${BASE_DIRECTORY}/generatebundlefile/bin/generatebundlefile  \
+    --input ${BASE_DIRECTORY}/generatebundlefile/data/staging_artifact_move.yaml.yaml \
+    --private-profile ${PROFILE}
+
+# Release Helm Chart, and bundle to Production account
+cat << EOF > stagingconfigfile
+[profile staging]
+role_arn=$ARTIFACT_DEPLOYMENT_ROLE
+region=us-east-1
+credential_source=EcsContainer
+EOF
+
+export AWS_CONFIG_FILE=${BASE_DIRECTORY}/generatebundlefile/stagingconfigfile
+export PROFILE=staging
+. "${BASE_DIRECTORY}/generatebundlefile/hack/common.sh"
+ECR_PUBLIC=$(aws ecr-public --region us-east-1 describe-registries --profile ${PROFILE} --query 'registries[*].registryUri' --output text)
+REPO=${ECR_PUBLIC}/eks-anywhere-packages-bundles
+
+aws ecr-public get-login-password --region us-east-1 | HELM_EXPERIMENTAL_OCI=1 helm registry login --username AWS --password-stdin public.ecr.aws
+
+# Move Helm charts within the bundle to another account
+${BASE_DIRECTORY}/generatebundlefile/bin/generatebundlefile  \
+    --input ${BASE_DIRECTORY}/generatebundlefile/data/staging_artifact_move.yaml.yaml \
+    --public-profile ${PROFILE}
+
+if [ ! -x "${ORAS_BIN}" ]; then
+    make oras-install
+fi
+
+function generate () {
+    local version=$1
+    local kms_key=signingPackagesKey
+
+    cd "${BASE_DIRECTORY}/generatebundlefile"
+    ./bin/generatebundlefile --input "./data/bundles_staging/${version}.yaml" \
+                 --key alias/${kms_key} \
+                 --output "output-${version}"
+}
+
+function push () {
+    local version=${1?:no version specified}
+    cd "${BASE_DIRECTORY}/generatebundlefile/output-${version}"
+    awsAuth "$REPO" | "$ORAS_BIN" login "$REPO" --username AWS --password-stdin
+    "$ORAS_BIN" push "${REPO}:v${version}-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
+    "$ORAS_BIN" push "${REPO}:v${version}-latest" bundle.yaml
+}
+
+for version in 1-20 1-21 1-22 1-23 1-24; do
+    generate ${version}
+done
+
+export AWS_PROFILE=prod
```

Comment:
Is prod the correct profile here? If so, a comment might be warranted.

Reply:
Fixed